src/controllers/updates_controller.js

```diff
@@ -104,10 +104,14 @@ UpdatesController.get('/status', async function(request, response) {
 });
 
 UpdatesController.post('/update', async function(request, response) {
-  const child = childProcess.spawn('node', ['tools/check-for-update.js'], {
-    detached: true,
-    stdio: 'ignore'
-  });
+  const child = childProcess.spawn(
+    process.argv[0],
```

Comment:
If a relative path is used, then we should probably also save away the current working directory at the time of program launch, or save away an absolute version of argv[0] at program start (just in case the current directory gets changed).

Reply:
`process.argv[0]` is secretly [process.execPath](https://nodejs.org/api/process.html#process_process_execpath) which is thankfully always absolute.